doc/prototype/filter-members.md

```diff
@@ -5,46 +5,64 @@
 
 @signature `queryLogic.filterMembers(a, [b,] bData)`
 
-Filters the `a` query's records from another _super set_ query's records.
-
-```js
-queryLogic.filterMembers(
-  {filter: {type: "dog"}},
-  {filter: {type: ["dog", "cat"]}},
-  [{id: 1, type:"cat"},
```

Comment:
it would be good to indent at least two spaces.

I thought we are using tabs for most of our docs?  What have you been using?

Reply:
We do indent two spaces, so I'm not sure what you're referring to here.

If tabs are used it defaults to an example with no syntax highlighting. 

#### Using spaces:
<img width="1291" alt="screen shot 2018-09-25 at 2 05 35 pm" src="https://user-images.githubusercontent.com/12449668/46043193-392e5800-c0cc-11e8-8181-7b166ffa9083.png">

#### Using _any_ tabs:
<img width="1278" alt="screen shot 2018-09-25 at 2 08 39 pm" src="https://user-images.githubusercontent.com/12449668/46043340-988c6800-c0cc-11e8-9b8e-1600ab5023fa.png">